packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -268,8 +269,17 @@ const App = (props) => {
         history.push(path)
     }
 
+    // Storefront Preview script. Your storefront needs to load this script for Storefront Preview feature
+    const storefrontPreviewClientScript =
+        process.env.NODE_ENV === 'development'
```

Comment:
`process.env` does not exist on client side

Reply:
it kinda does. Screenshot was for localhost:3000 storefront on browser
<img width="1004" alt="image" src="https://github.com/SalesforceCommerceCloud/pwa-kit/assets/52219283/60881a54-14b6-4a94-a029-b901ec324716">

Reply:
Hmm interesting.. maybe webpack automatically convert `process.env.NODE_ENV`.

Reply:
🤯